docs.old/requirements.txt

```diff
@@ -0,0 +1 @@
+mkdocs==1.1.2
```

Comment:
Probably safe to just delete`docs.old`, you can rely on `git` to remember how it was, no need to keep it around unless it's actually used - but then the `old` name is kind of confusing.